vmslave-config/ansible/group_vars/deb

```diff
@@ -1,2 +1,2 @@
 # on-imagebuilder is not working in ansible more than 2.0
-slave_ansible_version: 2.0.2
\ No newline at end of file
+ansible_version: 2.0.2
```

Comment:
nit: missing newline

Also, v-env ansible is set to 2.3, while this is set to 2.0.2. Should we be rolling these both forward to the same thing (2.3.0 or even 2.3.2(latest))?